cmd/nsm-init/Gopkg.toml

```diff
@@ -0,0 +1,50 @@
+# Gopkg.toml example
```

Comment:
Why are we creating a new Gopkg.toml? We should centralize everything in the top level Gopkg.toml.